src/migrate_openacc_2_openmp_convert.py

```diff
@@ -1337,10 +1336,10 @@ def translate (txConfig, lines, construct):
 	# information is needed later
 	carryOnStatus = dict ()
 	SupplementaryConstructs = dict()
-	 
-	for line, construct in construct.items():
+
+	for line, con in construct.items():
```

Comment:
I choose to shorten the variable name to resolve the reusing/overriding of the `construct` variable if it will be used afterwards. Using the same variable name twice for different purposes is a bad coding style in my mind.